inventory.pvfs

```diff
@@ -0,0 +1,21 @@
+# Ansible Shade uses OpenStack clients running locally
```

Comment:
Inventory files should be auto-generated by the Heat role (which you didn't run, so that's fine).

I'm wondering where your DNS is coming from, have you tinkered with that?

What's interesting here is that we have overlapping roles: a node can be both server and client.  This needs a little bit of thinking (but probably, in the first instance, manual adjustment)